### DISCUSSION TITLE
Version, Date and Author values not coming up on npm search

### DISCUSSION THREAD

**Discussion by shivajivarma** [verdaccio/verdaccio#2388] [Q&A] [answered]
When performing npm search on verdaccio server, it does not return version, date and author. Is there something i need to configure to make it work or is this a bug?

![Screenshot 2021-08-24 230132](https://user-images.githubusercontent.com/1388742/130662927-c4124e54-f918-4a94-985a-1c8447b3277d.png)

**Comment by juanpicado** [comment, marked as answer]
That my depends which npm version and which verdaccio version you are using.

Anyway I am working in a new `npm search ` so next major you will have a better experience.

https://github.com/verdaccio/verdaccio/pull/2340

**Reply by shivajivarma** [reply]
Here are my environment details:
NPM local machine: 6.14.8
verdaccio: 5.1.2

Thanks for quick response. Will follow the PR.

**Reply by juanpicado** [reply]
`npm@6.14.8` uses two version of npm search, not sure what you are hitting, if is undefined might be because the author is empty or malformed on the `package.json` of each package or a bug, if is a bug won't be fixed in v5, only in next major possibly v6.
